ui/src/main/webapp/src/app/project/migration-project-form.component.html

```diff
@@ -1,6 +1,31 @@
 <div class="container-fluid">
     <form #projectForm="ngForm" (ngSubmit)="save()" class="form-horizontal wizard-form">
 
+        <style>
```

Comment:
IDK, it's descriptive style, and may be actually better if applied well. But I understand this team doesn't prefer it.